qa/suites/fs/volumes/tasks/volumes/test/per_module_finisher_stats.yaml

```diff
@@ -0,0 +1,18 @@
+tasks:
+  - check-counter:
+      counters:
+        mgr:
```

Comment:
@vshankar @batrick Any idea how can I make this run only on active `mgr`. This config is making it run on all the mgrs running including active and standby. It's failing on standby, as the modules are not loaded in standby.

Reply:
Hmmm.. check_counter does try to get from all daemons. Maybe teach the task to only fetch from active daemons or run only a single instance of ceph-mgr?